auth0/v3/exceptions.py

```diff
@@ -6,3 +6,11 @@ def __init__(self, status_code, error_code, message):
 
     def __str__(self):
         return '{}: {}'.format(self.status_code, self.message)
+
+
+class TokenValidationError(Exception):
```

Comment:
doesn't extend `Auth0Error` because that one requires a status code (HTTP)

Reply:
If you just need the message attribute 

```python
class TokenValidationError(Exception):
  pass
```